### PR TITLE
cabal file

### PR DESCRIPTION
**Pull request by madjestic**
Hello,

I added a cabal file to build the project with kosher cabal.  Would you consider merging it?

Cheers,
Vlad

### CODE REVIEW

File: coinbase-pro.cabal [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+build-type:     Simple
+cabal-version:  >=2.0               
+                
+extra-source-files:
+    README.md
+    changelog.md
+
+source-repository head
+  type: git
+  location: https://github.com/https://github.com/mdunnio/coinbase-pro
```
Comment (mdunnio) [inline coinbase-pro.cabal]: This url is invalid.

**Comment by mdunnio** [comment]
Yeah, I don’t mind! Just fix up that one comment I can merge it.

File: coinbase-pro.cabal (hunk 1 above)
Comment (madjestic) [reply coinbase-pro.cabal]: no idea how that happened... Thanks!

Comment (madjestic) [reply coinbase-pro.cabal]: fixed!

**Comment by madjestic** [comment]
That should be fixed now.

**Review by mdunnio — APPROVED** [review]

**Pull request merged**